app/src/main/java/ch/epfl/sweng/erpa/CreateGame.java

```diff
@@ -0,0 +1,13 @@
+package ch.epfl.sweng.erpa;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class CreateGame extends AppCompatActivity {
```

Comment:
Can you please use android.app.Activity instead of android.support.v7.app.AppCompatActivity?